dvc/commands/ls_url.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import logging
+
+from dvc.cli.command import CmdBaseNoRepo
+from dvc.cli.utils import append_doc_link
+from dvc.exceptions import DvcException
+from dvc.ui import ui
+
+from .ls import _prettify
+
+logger = logging.getLogger(__name__)
+
+
+class CmdListUrl(CmdBaseNoRepo):
+    def run(self):
+        from dvc.repo import Repo
+
+        try:
+            entries = Repo.ls_url(self.args.url, recursive=self.args.recursive)
+            if entries:
+                entries = _prettify(entries, with_color=True)
+                ui.write("\n".join(entries))
+            return 0
+        except DvcException:
+            logger.exception("failed to list '%s'", self.args.url)
```

Comment:
Side note: another inconsistency in our code base, where we have these captain obvious messages in some commands but removed them long ago in others. No action needed for now, just ranting :)

Reply:
Yeah, this is pointless. Generic CLI error handling is fine - though we end up with an exit status of 255 instead of 1. The error message looks better now:

```
(dvc-dev) ronan@Creizic dvc % dvc ls-url foo
ERROR: The path 'foo' does not exist: [Errno 2] No such file or directory: '/Users/ronan/devel/iterative/dvc/foo'
```